commons-test/src/main/java/org/infinispan/commons/test/TestNGTestListener.java

```diff
@@ -20,6 +20,7 @@
    private final TestSuiteProgress progressLogger;
 
    public TestNGTestListener() {
+      EnvironmentCheck.checkJVMVersion();
```

Comment:
For TestNG modules we can throw an exception in `TestNGSuiteChecksTest.beforeSuite()` instead of calling `System.exit()`

Reply:
ok

Reply:
For JUnit I've moved the check to the testRunStarted method